publish-cli/pom.xml

```diff
@@ -136,8 +136,7 @@
                 <artifactId>spring-boot-maven-plugin</artifactId>
                 <configuration>
                     <mainClass>com.ericsson.eiffel.remrem.publish.cli.CLI</mainClass>
-                    <fork>true</fork>
-                    <skip>false</skip>
+                    <skip>true</skip>
```

Comment:
This need to be skipped, if not, when running mvn spring-boot:run for publish-service,, cli is executed instead of publish-service application. Thats why its skipped here.
For executing cli:
mvn package -DskipTests
java -jar target/cli.war